InvenTree/common/models.py

```diff
@@ -977,13 +976,6 @@ def save(self, *args, **kwargs):
             'validator': bool,
             'requires_restart': True,
         },
-        'ENABLE_PLUGINS_GLOBALSETTING': {
```

Comment:
@SchrodingersGat do you want plugins to have the right to define settings without enabling it first?

You will need to change this registry to, it checks if the setting is enabled:
 
 ​        ​if​ ​settings​.​PLUGIN_TESTING​ ​or​ ​InvenTreeSetting​.​get_setting​(​'ENABLE_PLUGINS_GLOBALSETTING'​):

Reply:
Now that the settings are quarantined in their own table, I don't think this is required - it is perhaps an extra impediment to configuration?

Reply:
How should plugins share configs like website-endpoints etc. then? My idea is that if a plugin already exists for an integration with another service than a new plugin can extend on the functions of the first plugin an reuse settings set for it.
Do you want another mixin for marking settings globally available?

Reply:
good point, I had not considered that... Although there's nothing stopping a plugin for looking at settings provided by *other* plugins (just be key lookup)...

Reply:
I think we will see in practice if that usecase pops up often enough to warrant a change / mixin.
I will rewrite my already existing plugins to use the new APIs and see if problems / code duplications arise.

Reply:
Fair enough, let's see how it plays out and refactor as necessary.